extensions/applicationinsights-clickanalytics-js/src/ClickAnalyticsPlugin.ts

```diff
@@ -153,22 +153,15 @@ export class ClickAnalyticsPlugin extends BaseTelemetryPlugin {
                     _autoCapture = autoCapture;
                 }));
             }
-        });
 
-        function _initDefaults() {
-            _config = null;
-            _pageAction = null;
-            _autoCaptureHandler = null;
-            _contentHandler = null;
-            _autoCapture = false;
-
-            // Define _self.config
-            objDefineProp(self, "config", {
```

Comment:
This was actually adding a `config` property to `window`...
Note: the `self` is missing the `_` prefix and even if it had that it's outside of the closure that defines `_self`...
So just removing as it's not used (or valid) to populate `window.config`